src/kpiUtils.ts

```diff
@@ -0,0 +1,5 @@
+export function getKpiStatus(numberPurchase: number): string {
```

Comment:
**Might have:** Maybe we can return directly type `KpiStatus` instead of `string`.

Create a new type in models with `KpiStatus`, `Enum` with values (`occasional`, `regular`, `VIP`)